<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>课堂签到管理系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>900</width>
     <height>500</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="4,2" columnminimumwidth="0,0">
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>     学生签到情况</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="tabStopWidth">
           <number>80</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>     学生人脸信息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_2">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>480</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>时间</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>日期</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>929</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>签到</string>
    </property>
    <addaction name="actionopen"/>
    <addaction name="actionclose"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>用户组</string>
    </property>
    <addaction name="actionaddgroup"/>
    <addaction name="actiondelgroup"/>
    <addaction name="actiongetlist"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>用户</string>
    </property>
    <addaction name="actionadduser"/>
    <addaction name="actiondeluser"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionopen">
   <property name="text">
    <string>启动签到</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="text">
    <string>关闭签到</string>
   </property>
  </action>
  <action name="actionaddgroup">
   <property name="text">
    <string>添加用户组</string>
   </property>
  </action>
  <action name="actiondelgroup">
   <property name="text">
    <string>删除用户组</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>存储信息</string>
   </property>
  </action>
  <action name="actiond">
   <property name="text">
    <string>导出信息</string>
   </property>
  </action>
  <action name="actiongetlist">
   <property name="text">
    <string>查询用户组</string>
   </property>
  </action>
  <action name="actionadduser1">
   <property name="text">
    <string>添加用户</string>
   </property>
  </action>
  <action name="actionadduser">
   <property name="text">
    <string>添加用户</string>
   </property>
  </action>
  <action name="actiondeluser">
   <property name="text">
    <string>删除用户</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
